### DISCUSSION TITLE
IS KEDA supports NATS messaging system?

### DISCUSSION THREAD

**Discussion by sachusachin622** [kedacore/keda#2789] [Q&A / Need Help] [answered]
Hello everyone!
Is it KEDA supports NATS messaging system [https://docs.nats.io/nats-concepts/what-is-nats](url) ?
Please help me to confirm.

Thank you!

**Comment by zroubalik** [comment, marked as answer]
https://keda.sh/docs/2.6/scalers/nats-streaming/
